src/content/docs/en/reference/experimental-flags/sessions.mdx

```diff
@@ -22,83 +22,151 @@ This experimental feature allows you to store and access items such as login sta
 
 :::note
-Some drivers may need extra packages to be installed. For example, the `netlify-blobs` driver requires the `@netlify/blobs` package, and Upstash requires the `@upstash/redis` package. Some drivers may also require environment variables or credentials to be set.
+Some drivers may need extra packages to be installed. For example, Upstash requires the `@upstash/redis` package. Some drivers may also require environment variables or credentials to be set.
+
+<ReadMore>
+  See [the unstorage driver docs](https://unstorage.unjs.io/drivers) for more information.
+</ReadMore>
 
-<ReadMore>See [the unstorage driver docs](https://unstorage.unjs.io/drivers) for more information.</ReadMore>
 :::
 
+Set the driver name in the `session.driver` property. You can find the driver name in the unstorage documentation for the driver you want to use.
+
+```js title="astro.config.mjs" ins={4}
+  {
+    adapter: vercel(),
+    session: {
+      driver: "upstash",
+    },
+    experimental: {
+      session: true,
+    },
+  }
+```
+
 ### Driver options
 
-You can also pass any available options to the unstorage driver in a separate `session.options` object. The following example configures the [Netlify Blobs](https://unstorage.unjs.io/drivers/netlify) driver while also providing a `name` and specifying a `consistency` mode:
+You can also pass any available options to the unstorage driver in a separate `session.options` object. See the driver documentation for available options. For example, to set the base key for sessions in Upstash:
 
-```js title="astro.config.mjs" ins={7-11}
+```js title="astro.config.mjs" ins={5-7}
   {
-    adapter: netlify(),
-    experimental: {
-      session: {
-        // The name of the unstorage driver is kebab-case
-        driver: "netlify-blobs",
-        options: {
-          name: 'astro-sessions',
-          // Sessions need strong consistency
-          consistency: 'strong',
-        }
+    adapter: vercel(),
+    session: {
+      driver: "upstash",
+      options: {
+        base: "sessions",
       },
     },
+    experimental: {
+      session: true,
+    },
+  }
+```
+
+### Other session options
+
+You can configure additional options for sessions in the `session` object. 
+
+#### `session.cookie`
+
+<p>
+
+**Type:** `string` | `object`<br />
+**Default:** `astro-session`<br />
+
+</p>
+
+Configures the session cookie. This cookie is set in the response when a session is generated. No actual user data is stored in the cookie – just an ID that is used to identify a user's session. The `session.cookie` option can be used to set options for this cookie. You can either provide a `string`, which will be used as the cookie name, or an object which allows additional options: 
```

Comment:
This feels more convoluted than what was at the old "cookies" entry. (Although, I do like the first sentence as a descriptor of the option!) Just checking that this is an intentional change that you don't like what already existed, which I think felt clearer?

Reply:
The reason I changed it was because previously this was in a separate section about cookies, but I removed that because I was moving the docs for the option into a section with other options, and once that was gone the cookies section didn't make much sense to remain. I then pulled in part of the wording from the deleted section.